extra_model/_cli.py

```diff
@@ -29,3 +29,7 @@ def entrypoint(input_path: Path, output_path: Path, debug: bool = False) -> None
     except ExtraModelError as e:
         logger.exception(e) if debug else logger.error(e)
         sys.exit(1)
+
```

Comment:
if we set up the CLI the right way this should not be necessary. The CLI wasn't being installed in the dev image so the `extra-model` entrypoint was unavailable, i am putting up a fix for this now 👍